node/src/app/cli.rs

```diff
@@ -21,6 +21,9 @@ use casper_node::{
 };
 use prometheus::Registry;
 
+#[global_allocator]
+static ALLOC: jemallocator::Jemalloc = jemallocator::Jemalloc;
```

Comment:
It would be nice to have a little comment on why we decided to swap out the allocator :)